src/style.css

```diff
@@ -0,0 +1,41 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Only `box-sizing` property allows for `*` selector

Reply:
Why?

Reply:
Not fixed